src/main/java/com/example/event_manager/service/TaskStatusServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package com.example.event_manager.service;
+
+import com.example.event_manager.model.TaskStatus;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+import java.util.List;
+
+public class TaskStatusServiceImpl implements TaskStatusService {
+  @Override
+  public boolean save(final TaskStatus taskStatus) throws NotImplementedException {
+    return false;
+  }
+
+  @Override
+  public void delete(final Long id) throws NotImplementedException {
+  }
+
+  @Override
+  public List<TaskStatus> findAll() throws NotImplementedException {
+    return null;
```

Comment:
if it throws then thow this exception in method implementation - in other case it make no sense to add throws to signature